### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by srandall97**
I added support for Docker and also added some documentation about setting up a database and Quill for production. I kept the documentation as general as possible and tried to mainly rely on links to other sources. I wasn't sure whether to provide command line instructions for Docker but I have tentatively decided to not include them.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM node:10-alpine
+WORKDIR /usr/app/src
+COPY . .
+EXPOSE 3000
+RUN apk add --no-cache --virtual .gyp python make g++
```
Comment (krubenok) [inline Dockerfile]: Do we need to this now that the dependency has been updated/removed?

**Review by krubenok — COMMENTED** [review]
Looks good. Need to get back to my computer and run this first. Will check back in on this tomorrow AM.

File: Dockerfile (hunk 1 above)
Comment (srandall97) [reply Dockerfile]: No I guess we don't need it anymore

**Comment by srandall97** [comment]
There were some weird diffs with the package-lock.json so I merged master -> docker and ran a check to make sure everything seems to be working. Looks good now so I'll merge.

**Pull request merged**